core/zms/src/main/rdl/Template.tdl

```diff
@@ -7,11 +7,22 @@ include "Role.tdl";
 include "Policy.tdl";
 include "ServiceIdentity.tdl"
 
+//MetaData for template.
+type MetaData Struct {
+    String description (optional); //description of the template
+    Int32 currentVersion (optional); //Version from DB(zms_store->domain_template->version)
+    Int32 updatedVersion (optional); //Bumped up version from solutions-template.json when there is a change
```

Comment:
instead of updatedVersion, let's call this latestVersion.

Reply:
Done. changed and checked in